vembrane/errors.py

```diff
@@ -13,7 +13,7 @@ def __str__(self):
 class UnknownAnnotation(VembraneError):
     """Unknown annotation entry"""
 
-    def __init__(self, record_idx: int, record: VariantRecord, key: str):
+    def __init__(self, record_idx: int, record, key: str):
```

Comment:
This (and all other methods below) are missing the type annotation for `record` (which I guess has to do with the import at line 3 is commented out?)

Reply:
again circular import, I don't know how to handle